src/gi/overrides/Vast.py

```diff
@@ -0,0 +1,6 @@
+from ..module import get_introspection_module
+
+Vast = get_introspection_module('Vast')
+
+Vast.Array.__getitem__ = lambda self, i: self.get_value(i)
```

Comment:
Sure, we need to make a call based on the Python datatype, but for now we need to add the corresponding APIs in the array.